test/RecipeRepository-test.js

```diff
@@ -1,8 +1,146 @@
+        "number": 1
+      },
+      {
+        "instruction": "Add egg and vanilla and mix until combined.",
+        "number": 2
+      },
+      {
+        "instruction": "Add dry ingredients and mix on low just until incorporated. Stir in chocolate chips.Scoop the dough into 1,5 tablespoon size balls and place on a plate or sheet. Cover with saran wrap and chill at least 2 hours or overnight.When ready to bake, preheat oven to 350 degrees.",
+        "number": 3
+      },
+    ],
+    "name": "Loaded Chocolate Chip Pudding Cookie Cups",
+    "tags": [
+      "antipasti",
+      "starter",
+      "snack",
+      "appetizer",
+      "antipasto",
+      "hor d'oeuvre"
+    ]
+  },
+  {
+    "id": 678353,
+    "image": "https://spoonacular.com/recipeImages/678353-556x370.jpg",
+    "ingredients": [
+      {
+        "id": 1009016,
+        "quantity": {
+          "amount": 1.5,
+          "unit": "cups"
+        }
+      },
+      {
+        "id": 9003,
+        "quantity": {
+          "amount": 2,
+          "unit": ""
+        }
+      },
+      {
+        "id": 20027,
+        "quantity": {
+          "amount": 1,
+          "unit": "tablespoon"
+        }
+      },
+    ],
+    "instructions": [
+      {
+        "instruction": "Season the pork chops with salt and pepper and grill or pan fry over medium high heat until cooked, about 3-5 minutes per side. (If grilling, baste the chops in the maple dijon apple cider sauce as you grill.)Meanwhile, mix the remaining ingredients except the apple slices, bring to a simmer and cook until the sauce thickens, about 2-5 minutes.Grill or saute the apple slices until just tender but still crisp.Toss the pork chops and apple slices in the maple dijon apple cider sauce and enjoy!",
+        "number": 1
+      }
+    ],
+    "name": "Maple Dijon Apple Cider Grilled Pork Chops",
+    "tags": [
+      "lunch",
+      "main course",
+      "main dish",
+      "dinner"
+    ]
+  },
+];
+
+describe('Recipe Repository', () => {
+  let repo;
+
+  beforeEach(() => {
+    repo = new RecipeRepository(recipeData);
+  });
 
-describe('Recipe', () => {
   it('Should be a function', () => {
     expect(RecipeRepository).to.be.a('function');
   });
-})
\ No newline at end of file
+  
+  it('Should be an instance of Recipe', () => {
+    expect(repo).to.be.an.instanceOf(RecipeRepository);
+  });
+
+  it('Should have recipes', () => {
+    expect(repo.recipes[0].id).to.equal(595736);
+  });
+
+  it('Should filter recipes based on one tag', () => {
+    const expected = repo.findRecipesByTag('snack');
+    expect(expected[0].name).to.equal('Loaded Chocolate Chip Pudding Cookie Cups');
+    expect(expected.length).to.equal(1);
+  });
+
+  it('Should filter recipes based on more than one tag', () => {
+    const expected = repo.findRecipesByTag('snack', 'antipastisi');
```

Comment:
It might make sense to make the tags an array of strings so you we can iterate through those... does that make sense?

Reply:
I think so! I will change that now!